packages/playwright-core/src/common/errors.ts

```diff
@@ -25,16 +25,13 @@ class CustomError extends Error {
 
 export class TimeoutError extends CustomError {}
 
-export const kTargetClosedErrorMessage = 'Target page, context or browser has been closed';
-export const kTargetCrashedErrorMessage = 'Target crashed';
-
 export class TargetClosedError extends Error {
```

Comment:
Should this be `extends CustomError` similarly to `TimeoutError`?

Reply:
I have no idea!